doc/fuzzing.md

```diff
@@ -7,11 +7,7 @@ To quickly get started fuzzing Bitcoin Core using [libFuzzer](https://llvm.org/d
 ```sh
 $ git clone https://github.com/bitcoin/bitcoin
 $ cd bitcoin/
-$ cmake -B build_fuzz \
-   -DCMAKE_C_COMPILER="clang" \
-   -DCMAKE_CXX_COMPILER="clang++" \
-   -DBUILD_FOR_FUZZING=ON \
-   -DSANITIZERS=undefined,address,fuzzer
+$ cmake --preset=libfuzzer
```

Comment:
-nosan option imo is worth noting directly here since it's slightly less clear what I could remove to make things faster